### PR TITLE
build: Update internals to no longer use deprecated `Decorated`

### PR DESCRIPTION
**Pull request by emostov**
Closes @293 

Changes:
 - `createDecorated` becomes `createDecoratedTx` and `createDecoratedConstants`
 - update `createDecorated` users to reflect changes
 - bump polkadot-js/api to  2.4.2-13

See #293 for more info

### CODE REVIEW

File: src/util/metadata.ts [hunk 1]
```diff
@@ -73,18 +78,32 @@ export const createMetadata = memoizee(createMetadataUnmemoized, {
...
+}
+
+/**
+ * From a metadata hex string (for example returned by RPC), create decorated
+ * modules with their constants.
+ *
+ * @param registry - The registry of the metadata.
+ * @param metadata - The metadata as hex string.
+ */
+export function createDecoratedConststants(
```
Comment (insipx) [inline src/util/metadata.ts]:
```suggestion
export function createDecoratedConstants(
```

File: src/util/claims.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { StatementKind } from '@polkadot/types/interfaces';
 import { u8aToHex, u8aToString } from '@polkadot/util';
 import { decodeAddress } from '@polkadot/util-crypto';
 
-import { createDecorated } from './metadata';
+import { createDecoratedConststants } from './metadata';
```
Comment (insipx) [inline src/util/claims.ts]:
```suggestion
import { createDecoratedConstants } from './metadata';
```

File: src/util/claims.ts [hunk 3]
```diff
@@ -63,8 +63,11 @@ export function getEthereumPayload(
   statement: PolkadotStatement,
   options: OptionsWithMeta
 ): string {
-  const decorated = createDecorated(options.registry, options.metadataRpc);
-  const prefix = u8aToString(decorated.consts.claims.prefix.toU8a(true));
+  const constants = createDecoratedConststants(
```
Comment (insipx) [inline src/util/claims.ts]:
```suggestion
  const constants = createDecoratedConstants(
```

**Review by insipx — APPROVED** [review]
LGTM other than the typos

**Review by dvdplm — APPROVED** [review]

**Pull request merged**